### ISSUE TITLE
Odd arrangement of HTML elements in third-party app info bullet list

### ISSUE THREAD

**Issue by SabreCat** [HabitRPG/habitica#12698]
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)

As pointed out by @citrusella:

The nesting of HTML elements in `website/client/src/components/settings/api.vue` is wonky. Ideally, for the sake of accessibility and readability of the markup itself, each bullet point in a list should be contained in a `<li>` element. But on the section of the API Settings where we list information about some known third-party apps, there's a mix of `<li>`s and `<span>`s that break up the flow.

This section should be rearranged so that each bullet is a single `<li>` with any necessary line breaks etc. nested within it, instead of as siblings to it.

**Comment by agarwalvaibhav0211** [comment]
@SabreCat I would like to work on this Issue.

**Comment by Alys** [comment]
@agarwalvaibhav0211 Thank you! Please go ahead! Post here if you have questions.

**Comment by agarwalvaibhav0211** [comment]
@Alys please check PR #12732
